### ISSUE TITLE
Text Wrapping Hyphenation

### ISSUE THREAD

**Issue by AlbinoGeek** [fyne-io/fyne#1660]
TextProvider should offer **optional** text wrapping hyphenation along with its existing wrapping features.

Why optional? This feels like it should be another wrapping mode "hyphen-word" v.s "whole-word"

Related: #332

**Comment by andydotxyz** [comment]
Indeed. We envisaged that this would be added to the list of truncate/wrap types - just did not have time to implement it when adding wrapping.
